lib/screens/v2/widget_instance/alert.ex

```diff
@@ -355,6 +355,23 @@ defmodule Screens.V2.WidgetInstance.Alert do
     MapSet.subset?(active_routes_at_stop(t), BaseAlert.informed_routes_at_home_stop(t))
   end
 
+  defp informed_routes(%__MODULE__{screen: %Screen{app_id: :gl_eink_v2}} = t) do
```

Comment:
Do we need a test for this new def?

Reply:
I added some comments to help with clarity. Let me know if it still isn't clear. I also added some tests addressing a few different alert scenarios,

Reply:
Thanks for the comments! Seems like we should eventually refactor so that all screens use a function like the new one you added here, and scrap the old one. Because in all cases, we'd rather get affected routes from the alert than from the stop, right?

Reply:
Yeah that would make the most sense to me, but with context of the screen location in mind. For example, a Gov't Center closure would show on some of our GL E-Ink screens, but we don't care about BL on those screens. If we were to put this widget on other subway screens, we would need to consider whether that would be valid logic for those screens as well. Showing pills for all affected routes in the alert without filtering could cause us to give the rider info they don't need.